content/en/docs/user-journeys/users/application-developer/foundational.md

```diff
@@ -53,6 +53,33 @@ Minikube can be installed locally, and runs a simple, single-node Kubernetes clu
 
 You can get basic information about your cluster with the commands `kubectl cluster-info` and `kubectl get nodes`. However, to get a good idea of what's really going on, you need to deploy an application to your cluster. This is covered in the next section.
 
+#### microk8s
+
+On Linux, *microk8s* might be a good alternative to Minikube for a
```

Comment:
I still don't know how to evaluate whether it "might" be a good use case for me or not. Can you give some criteria for deciding?

Reply:
I intended to elaborate on that statement a little further in the same paragraph. For example, microk8s is a great alternative to Minikube if you do not want the overhead of running a VM. Does that make sense?